firebase-dataconnect/src/androidTest/kotlin/com/google/firebase/dataconnect/GrpcMetadataIntegrationTest.kt

```diff
@@ -303,6 +372,20 @@ class GrpcMetadataIntegrationTest : DataConnectIntegrationTestBase() {
     val googApiClientHeader: Metadata.Key<String> =
       Metadata.Key.of("x-goog-api-client", Metadata.ASCII_STRING_MARSHALLER)
 
+    fun expectedGoogApiClientHeader(isFromGeneratedSdk: Boolean) = buildString {
+      append("gl-kotlin/${KotlinVersion.CURRENT}")
+      append(' ')
+      append("gl-android/${Build.VERSION.SDK_INT}")
+      append(' ')
+      append("fire/${BuildConfig.VERSION_NAME}")
+      append(' ')
+      append("grpc/")
```

Comment:
Is there supposed to be a grpc version here?

Reply:
Sadly, no, there is no reliable way to determine the grpc version. I wasn't sure what to do here so I just copied Firestore's logic, which just omits the version number:

https://github.com/firebase/firebase-android-sdk/blob/fb2f8b24afc581dbb22d453859bbfe8bb37d4caa/firebase-firestore/src/main/java/com/google/firebase/firestore/remote/FirestoreChannel.java#L318